nixos/modules/services/network-filesystems/glusterfs.nix

```diff
@@ -5,6 +5,22 @@ with lib;
 let
   inherit (pkgs) glusterfs rsync;
 
+  tlsCmd = if cfg.enableTLS then
+  ''
+    mkdir -p /var/lib/glusterd
+    touch /var/lib/glusterd/secure-access
+  ''
+  else
+  ''
+    rm -f /var/lib/glusterd/secure-access
```

Comment:
If we don't want to add the keys to the nixstore and don't manipulate the file in `/var/lib/glusterd` then there is actually nothing much this change does. As anyway everything (put keys somewhere, put activation file in `/var`) has to be done outside of nix?